eu.usdx.UltraStarDeluxe.yaml

```diff
@@ -0,0 +1,194 @@
+id: eu.usdx.UltraStarDeluxe
+branch: master
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '22.08'
+separate-locales: false
+appstream-compose: true
+rename-desktop-file: ultrastardx.desktop
+rename-icon: ultrastardx
+command: ultrastardx
+
+finish-args:
+- --socket=wayland
```

Comment:
does it really support wayland?

Reply:
It supports everything supported by SDL2. I ran it on Wayland once.

Reply:
so you don't know.

BTW "running under wayland" doesn't mean "support wayland" as by default wayland comes with Xwayland for apps that don't support wayland.

Reply:
SDL2 has a Wayland backend it that's what you are asking.